<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>convertDialog</class>
 <widget class="QDialog" name="convertDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Import DB</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="db_in"/>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QPushButton" name="selectdb">
     <property name="text">
      <string>Select DB</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Tables</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="tables"/>
   </item>
   <item row="1" column="2" colspan="3">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Out table name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Output DB</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="db_out"/>
   </item>
   <item row="2" column="2" colspan="4">
    <widget class="QLineEdit" name="db_out_table"/>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="QLabel" name="message">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" rowspan="7" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Fields</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="2">
       <widget class="QLabel" name="pil_5">
        <property name="text">
         <string>--&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="fieldLabel_2">
        <property name="text">
         <string>Field 2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="fieldLabel">
        <property name="text">
         <string>Field 1</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="fieldLabel_5">
        <property name="text">
         <string>Field 5</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QComboBox" name="utfield5"/>
      </item>
      <item row="3" column="3">
       <widget class="QComboBox" name="utfield4"/>
      </item>
      <item row="2" column="3">
       <widget class="QComboBox" name="utfield3"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="infield2"/>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="pil_3">
        <property name="text">
         <string>--&gt;</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="infield6"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="fieldLabel_6">
        <property name="text">
         <string>Field 6</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="fieldLabel_4">
        <property name="text">
         <string>Field 4</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="infield3"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="pil_2">
        <property name="text">
         <string>--&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="utfield1"/>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="utfield2"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="pil">
        <property name="text">
         <string>--&gt;</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QComboBox" name="utfield6"/>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="infield4"/>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="pil_6">
        <property name="text">
         <string>--&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="infield1"/>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="infield5"/>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="pil_4">
        <property name="text">
         <string>--&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="fieldLabel_3">
        <property name="text">
         <string>Field 3</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>53</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="2" colspan="4">
    <widget class="QCheckBox" name="updateSettings">
     <property name="text">
      <string>Update settings</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Load fields</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2" colspan="4">
    <widget class="QLineEdit" name="addfieldname"/>
   </item>
   <item row="8" column="2">
    <widget class="QToolButton" name="addfield">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>+/- field</string>
     </property>
    </widget>
   </item>
   <item row="8" column="4">
    <widget class="QToolButton" name="removefield">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="8" column="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>5</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="2" colspan="2">
    <widget class="QPushButton" name="convert">
     <property name="text">
      <string>Convert</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2" colspan="2">
    <widget class="QPushButton" name="close">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>db_in</tabstop>
  <tabstop>selectdb</tabstop>
  <tabstop>tables</tabstop>
  <tabstop>db_out</tabstop>
  <tabstop>db_out_table</tabstop>
  <tabstop>infield1</tabstop>
  <tabstop>utfield1</tabstop>
  <tabstop>infield2</tabstop>
  <tabstop>utfield2</tabstop>
  <tabstop>infield3</tabstop>
  <tabstop>utfield3</tabstop>
  <tabstop>infield4</tabstop>
  <tabstop>utfield4</tabstop>
  <tabstop>infield5</tabstop>
  <tabstop>utfield5</tabstop>
  <tabstop>infield6</tabstop>
  <tabstop>utfield6</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>addfieldname</tabstop>
  <tabstop>addfield</tabstop>
  <tabstop>removefield</tabstop>
  <tabstop>convert</tabstop>
  <tabstop>close</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>convertDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
